docs/release.md

```diff
@@ -39,13 +42,26 @@ To create a new release, follow these steps:
 As part of the process, [release-it](https://www.npmjs.com/package/release-it) will
 perform a number of checks, and fail if anything isn't right. For example, your
 working directory must be clean (no local file changes), you must have an `upstream`
-remote, the tests must pass, etc.
-
-### Releases
+remote, the tests must pass, etc.\
 
 Once a release is complete, a new [git tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) will be created in the project's history. A new
 [GitHub Release will also be created](https://github.com/Seneca-CDOT/telescope/releases).
 
+### Using npm-version
+
+To create a new release, follow these steps:
+
+1. Make sure your `master` branch is up-to-date, and you have the most recent git tags in your repo: `git pull upstream master --tags`.
+1. Run `npm outdated` to see a report of [outdated npm packages](https://docs.npmjs.com/cli-commands/outdated.html). Examine this list and decide whether we need to update
+   anything now, or before the next release. _Please file a new Issue to update
+   anything that is outdated_.
+1. Make sure your working tree is clean.
+1. Determine what the new version _string_ should be based on [semantic versioning](https://github.com/npm/node-semver#functions). The `newversion` argument should be a valid semver string. In our project, that would usually be `minor` (e.g. `1.5.0`, `1.6.0`, `1.7.0`) or `major` (e.g. `1.0.0`, `2.0.0`). See [npm-version docs](https://docs.npmjs.com/cli/v6/commands/npm-version) to learn more about what options are available for the new version string.
+1. Use `npm version <new-version-string> -m "Release message` to trigger the automated release workflow.
+
+`npm-version` will proceed to run tests locally. If successful, it will also bump the `version` in [package.json](https://github.com/Seneca-CDOT/telescope/blob/master/package.json), create a new [`git tag`](https://git-scm.com/book/en/v2/Git-Basics-Tagging) and push both the code and the tags to `upstream master`.
```

Comment:
Add a note about how you must be an admin and must have an upstream pointing to the main Seneca CDOT telescope repo

Reply:
Updated the whole, added a note about this [here](https://github.com/Seneca-CDOT/telescope/pull/1650/files#diff-87479ac9ea16cec5ceafce00b3b049d7d5920cb663afae42901f20e7c4e6705dR11-R12)